src/utils/arrayUtils.ts

```diff
@@ -14,3 +14,14 @@ export function chunk<T>(array: T[], chunkSize: number): T[][] {
   // eslint-disable-next-line @typescript-eslint/no-unsafe-return
   return result
 }
+
+export async function callChunked<T>(
+  chunkSize: number,
+  array: readonly T[],
+  processFn: (arrayChunk: T[]) => Promise<void>,
```

Comment:
🔶 `callChunked` actually doesn't care about the callback's return type (other than it being awaitable).

We can loosen the type restriction therefore to:

```ts
  processFn: (arrayChunk: T[]) => Promise<unknown>
```

Reply:
fixed